python/afdko/otf2ttf.py

```diff
@@ -36,9 +40,12 @@ def glyphs_to_quadratic(
     return quadGlyphs
 
 
+@singledispatch
 def otf_to_ttf(ttFont, post_format=POST_FORMAT, **kwargs):
-    assert ttFont.sfntVersion == "OTTO"
-    assert "CFF " in ttFont
+    if ttFont.sfntVersion != "OTTO":
+        raise TTLibError("Not a OpenType font (bad sfntVersion)")
+    if "CFF " not in ttFont:
+        raise TTLibError("Not a OpenType font (no 'CFF ' table)")
```

Comment:
Are both of these checks required? I think only one of them is sufficient.

Reply:
I have ever considered one possibility what if a malicious program disguised as a font file with legal TTC header. The last important problem about font security that I have remembered is that Windows 10 change font kernel mode to user mode in order to alleviate relevant issues.

Reply:
OK. Doesn't hurt to leave both.

Reply:
But I prefer to recover initial single check if any maintenance burden is excessively added. Overall, I feel somehow nervous to add these enhancements if they add unnecessary complexity.

Reply:
But if you revert to the original single assert will the tool still gracefully skip TTFs, or will it halt with a traceback at the first TTF it finds?

Reply:
Do have no influence at all except that a non-otf file with legal otf header.

Reply:
OK, go ahead with the solution you think it's best.